ellipticoind/src/api/query_root.rs

```diff
@@ -88,8 +88,10 @@ impl QueryRoot {
                 let balance =
                     token::get_balance(&mut api, liquidity_token.clone(), address.clone());
                 let price = exchange::get_price(&mut api, token.clone()).unwrap_or(0);
-                let share_of_pool = exchange::share_of_pool(&mut api, token, address.clone());
+                let share_of_pool =
+                    exchange::share_of_pool(&mut api, token.clone(), address.clone());
                 let total_supply = token::get_total_supply(&mut api, liquidity_token.clone());
+                let total_pool_supply = exchange::get_reserves(&mut api, token.clone());
```

Comment:
A (somewhat pedantic) goal I have is to use the same terminology from the database to the frontend. Can we change `exchange::get_reserves` to `exchange::get_total_pool_supply` or `total_pool_supply` to `reserves`?